services/static-webserver/client/source/class/osparc/po/PreRegistration.js

```diff
@@ -17,54 +17,38 @@
 ************************************************************************ */
 
 qx.Class.define("osparc.po.PreRegistration", {
-  extend: qx.ui.core.Widget,
```

Comment:
you should have asked me to do this as an exercise :-)

Reply:
next time ;)